win/pkgs/vsbuildtools.ps1

```diff
@@ -0,0 +1,28 @@
+#Requires -RunAsAdministrator
+
+$ErrorActionPreference="Stop"
+
+$vs_where = "${Env:ProgramFiles(x86)}/Microsoft Visual Studio/Installer/vswhere.exe"
+if (Test-Path $vs_where)
+{
+    $vs_home = & $vs_where -latest -products * -property installationPath
+    if ($vs_home)
+    {
+        Write-Host "Found existing VS installtaion under `"$vs_home`". Skip installation."
+        exit 0
+    }
+}
+
+Invoke-WebRequest -Uri https://aka.ms/vs/16/release/vs_buildtools.exe -OutFile "${Env:SCRATCH}/vs_buildtools.exe"
+
+$install_cmd = "`"${Env:SCRATCH}/vs_buildtools.exe`" --quiet --wait --norestart --nocache " `
+    + "--add Microsoft.VisualStudio.Workload.VCTools;includeRecommended "                   `
+    + "--add Microsoft.VisualStudio.Component.Windows10SDK.16299 "                          `
+    + "--installPath C:\BuildTools || exit /b %ERRORLEVEL%"
+cmd /c $install_cmd
+
+if ($LASTEXITCODE -ne 0 -or $LASTEXITCODE -ne 3003)
```

Comment:
What is 3003, please explain in comment and link to doc if any.

Reply:
Ah, it should be 3010.

And to be honest, I don't know what 3003 stands for either. I saw it from several places:
- https://docs.microsoft.com/en-us/visualstudio/install/build-tools-container?view=vs-2019
- https://github.com/microsoft/vs-dockerfiles/blob/master/native-desktop/Install.cmd

Reply:
Found that in your linked page: 
> Error code 3010 is used to indicate success with a reboot required, see MsiExec.exe error messages for more information.

Please comment in code (in a few words), with the link as well.

Reply:
And 3010 is probably a failure in our case, since it is still not usable until rebooted.

Reply:
Make sense. Remove `3010` error code handling.